.github/ISSUE_TEMPLATE.md

```diff
@@ -1,37 +1,27 @@
 <!--
-Questions:
-  https://groups.google.com/forum/#!forum/loopbackjs
-  https://gitter.im/strongloop/loopback
-Immediate support:
-  https://strongloop.com/api-connect-faqs/
-  https://strongloop.com/node-js/subscription-plans/
+HELP US HELP YOU BY PROVIDING AS MUCH OF THE FOLLOWING INFORMATION AS POSSIBLE AND
+AVOID DUPLICATES BY SEARCHING BEFORE CREATING A NEW ISSUE.
 -->
 
-# Description/Steps to reproduce
+## I'm submitting a ...
+<!-- Check one of the following options with a "x" -->
+<pre><code>
+[ ] Regression (a behavior that used to work and stopped working in a new release)
+[ ] Bug report
+[ ] Feature request <!-- Include a problem statement, use case scenario, example, code samples, etc. -->
+[ ] Documentation issue or request
+[ ] Support / Question
+[ ] Feedback / Comment / Thanks / Other
+</code></pre>
 
-<!--
-If feature: A description of the feature
-If bug: Steps to reproduce
--->
-
-# Link to reproduction sandbox
```

Comment:
I think we eventually need to come up with a solution allowing people to provide us with a reproduction sandbox. I am thinking about adding a new monorepo package `sandbox`. This package should contain an empty app created via `lb4`. By keeping the sandbox in monorepo, people can easily run their code against the latest `master` branch (`lerna` will link dependencies from monorepo instead of installing from npmjs.org).

Anyhow, I think this is out of scope of this pull request.

Reply:
Neat idea! IIUC ... they would clone the entire monorepo to provide the sandbox which in turn makes it easier for us to debug and helps ensure it's being checked against the latest version.

Reply:
Yes, your understanding is correct - we will ask people to fork our entire monorepo, create a feature branch to demonstrate the problem and then give us a link to that feature branch.

> idk how users would feel about cloning the entire monorepo.

What are your concerns? In LoopBack 3.x, we were asking people to fork & clone our sandbox repo. I personally don't see much difference between forking and cloning a GitHub project with a single package vs. a monorepo with multiple packages.